python/cudf/cudf/core/series.py

```diff
@@ -134,6 +134,22 @@ def from_pandas(cls, s, nan_as_null=True):
 
     @property
     def values(self):
+        if not utils._have_cupy:
+            raise ModuleNotFoundError("cuPY was not found.")
+        import cupy
+
+        if is_categorical_dtype(self.dtype) or np.issubdtype(
+            self.dtype, np.dtype("object")
+        ):
+            raise TypeError("Data must be numeric")
+
+        if self.data.mem.size == 0:
+            return cupy.asarray([])
```

Comment:
You should type it the same as the Series.
```suggestion
            return cupy.asarray([], dtype=self.dtype)
```